addons/sourcemod/scripting/mce/cvars.inc

```diff
@@ -202,7 +210,11 @@ stock void CvarsInit()
 	g_Cvar_TimerUnlockNoms = CreateConVar("mce_locknominations_timer", "15.0", "Unlock nominations after a vote. Time in seconds.", _, true, 0.0, true, 60.0);
 	g_Cvar_ShowNominator = CreateConVar("mce_shownominator", "1", "See who nominated the map which won the vote", _, true, 0.0, true, 1.0);
 	g_Cvar_SharedCDMode = CreateConVar("mce_sharedcd_mode", "1", "Group maps with shared cooldowns mode (See ExcludeMap native docs for mode)", _, true, 0.0, true, 2.0);
-
+	g_Cvar_UseTiers = CreateConVar("mce_use_tiers", "1", "Use tiers for maps (map difficulty)", _, true, 0.0, true, 1.0);
+	g_Cvar_ShowMapTier = CreateConVar("mce_show_map_tier", "1", "See the tier of the map which won the vote [Dependecy: mce_use_tiers]", _, true, 0.0, true, 1.0);
+	g_Cvar_DefaultTier = CreateConVar("mce_default_tier", "1", "Default map tiers to set if tier param was not found", _, true, 1.0);
+	g_Cvar_MaxTiers = CreateConVar("sm_mce_max_tiers", "3", "How many tiers will be found in translation file.", _, true, 1.0, true, 10.0);
```

Comment:
Echo of my comment, but I don't think this cvar is necessary. Unless you plan on having custom names for tiers which would be an entirely different thing, I think you could just pass tiers directly from config to the plugin. Also, ideally we don't want to have users manually edit the translation files. Keep in mind that when we release the plugin, it needs to be in a complete state where managers can just change cvars and it will still work. The translation file you provided only goes up to T3, but what if server owners want up to T7 but the translation file isn't updated? IMO this whole convar setup needs to be reconsidered.